docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -501,6 +502,15 @@ it may be absolute or relative to the location of the unit file.
 
 This key may be used multiple times
 
+### `ExitPolicy=`
+
+Control how the main PID of the systemd service should exit.  The following values are supported:
+- `all`: exit non-zero if all containers have failed (i.e., exited non-zero)
+- `any`: exit non-zero if any container has failed
+- `none`: exit zero and ignore failed containers
```

Comment:
It may be too late for this, or there may be reasons why it's important to leave it this way, but I need to speak up: this name can be ambiguous and confusing. As a naïve human, I can interpret "ExitPolicy" as "exit success" or "exit failure" or simply "exit", and all/any/none do not give me an intuitive clue. I would strongly prefer a name that makes it 100% clear that this policy refers to the propagation of errors. Unfortunately I can't think of a good example, because naming is always the hardest part of programming. Naming is also the most permanent and most costly in the long term, so if anyone else agrees with my concern, can we brainstorm a little?

Reply:
Just being cheap and using the [thesaurus](https://www.thesaurus.com/browse/policy), maybe Action, Behavior, Method, Protocol, Strategy?

Reply:
I think `ExitCodePropagation`  matches closely what @edsantiago is looking for. @ygalblum WDYT?

Reply:
Yes @vrothberg, I like your suggestion.